test/runtests.jl

```diff
@@ -52,6 +52,7 @@ using SafeTestsets
     
     ### Tests extensions. ###
     if VERSION >= v"1.9"
```

Comment:
This can be dropped now once rebased since we only support 1.9 and up going forward.

Reply:
done